dockerfiles/Dockerfile.shellcheck

```
@@ -1,5 +1,9 @@
-FROM    koalaman/shellcheck-alpine:v0.7.1
-RUN     apk add --no-cache bash make
+# syntax=docker/dockerfile:1.3
+
+FROM koalaman/shellcheck-alpine:v0.7.1 AS shellcheck
 WORKDIR /go/src/github.com/docker/cli
-ENV     DISABLE_WARN_OUTSIDE_CONTAINER=1
-COPY    . .
+RUN apk add --no-cache bash
+RUN --mount=type=bind,target=. \
+  set -eo pipefail; \
+  shellcheck contrib/completion/bash/docker \
+  && find scripts/ -type f | grep -v scripts/winresources | grep -v '.*.ps1' | xargs shellcheck
```

We can combine this (add the `contrib/completion/bash` to the `find` command;

```suggestion
    set -eo pipefail; \
    find scripts/ contrib/completion/bash -type f | grep -v scripts/winresources | grep -v '.*.ps1' | xargs shellcheck
```